src/ApiRequestPanel.js

```diff
@@ -683,11 +737,21 @@ export class ApiRequestPanel extends AmfHelperMixin(EventsTargetMixin(HeadersPar
     this.updateServers({ id: selected, type, endpointId });
   }
 
+  _handleServersCountChange(e) {
+    const { serversCount } = e.detail;
+    this.serversCount = serversCount;
+  }
+
   _getServerUri(server) {
     const key = this._getAmfKey(this.ns.aml.vocabularies.core.urlTemplate);
     return this._getValue(server, key);
   }
 
+  _evaluateShowServerSelector() {
```

Comment:
Maybe this should be `_computeShowServerSelector` to maintain naming conventions

Reply:
Agree

Reply:
Or `_hasShowServerSelector`. Whatever fits you.